### PR TITLE
[stl] add standard text tags package

### PR DESCRIPTION
**Pull request by Cokemonkey11**
This suffers from a bit of code duplication but it is not too bad.

One potential improvement would be to extract the colors into
more reuseable vec3s in Colors package.

(not tested)

### CODE REVIEW

**Comment by Cokemonkey11** [comment]
@Frotty do you plan to review this?

File: Wurstpack/wurstscript/lib/small helpers/StandardTextTags.wurst [hunk 1]
```diff
@@ -0,0 +1,76 @@
+package StandardTextTags
+    constant real FONT_SIZE  =  0.024 / 0.0023
```
Comment (Frotty) [inline Wurstpack/wurstscript/lib/small helpers/StandardTextTags.wurst]: Value based on?

**Comment by Frotty** [comment]
There could probably be some preset function for texttags itself that these functions call so it's not the same extension funcs in every function.
Please add a one-line of hotdoc to each func then I will merge

File: Wurstpack/wurstscript/lib/small helpers/StandardTextTags.wurst (hunk 1 above)
Comment (Cokemonkey11) [inline Wurstpack/wurstscript/lib/small helpers/StandardTextTags.wurst]: http://www.wc3c.net/showthread.php?t=101963

**Comment by Cokemonkey11** [comment]
@Frotty okay, this should be good. Let me know if it's ready and I will squash.

**Comment by peq** [comment]
I put the StdLib into a separate repo, please create the same pull request over there:  https://github.com/peq/wurstStdlib

**Pull request closed without merging**